spec/view_helper_spec.rb

```diff
@@ -51,6 +51,21 @@
         expect(meta).to have_tag('meta', with: { content: "facebook description", property: "og:description" })
       end
     end
+
+    it 'should work with app links parameters' do
+      subject.set_meta_tags('al' => {
+        'ios' => {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.